pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -1,3 +1,11 @@
+{ buildPackages, runCommand, nettools, bc, perl, gmp, libmpc, mpfr, openssl
+, ncurses
+, libelf
+, utillinux
+, writeTextFile, ubootTools
+, callPackage
+}:
+
```

Comment:
I don't understand the intention here.  Most of the parameters are unused, and having `buildPackages` twice is a bit confusing...

Reply:
That's a good point. Sorry I missed this.

Reply:
I copy pasted from manual-config.nix and forgot to trim it down. Will do in https://github.com/NixOS/nixpkgs/pull/34351#discussion_r167437046 sorry

Reply:
When doing this, we might also clean up the parameters in `<nixpkgs/pkgs/os-specific/linux/kernel/linux-*.nix>` as well.